common/djangoapps/third_party_auth/tests/data/lti_garbage.txt

```diff
@@ -1 +1 @@
-some=garbage&values=provided
\ No newline at end of file
+some=garbage&values=provided&oauth_signature_method=HMAC-SHA1&oauth_signature=STPWUouDw%2FlRGD4giWf8lpGTc54%3D&oauth_timestamp=1436823553&oauth_version=1.0
```

Comment:
@nasthagiri I have updated test against it and added a new one. See if it makes sense.